src/main.cpp

```diff
@@ -6019,10 +6024,9 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
     else if (strCommand == NetMsgType::BLOCK && !fImporting && !fReindex) // Ignore blocks received while importing
     {
         CBlock block;
+        pfrom->nBlockSize = vRecv.size(); // Used by ProcessNewBlock() debug
```

Comment:
My question is more what do you want to log? How many bytes are being transferred, or the consensus-critical block size?

Assuming compact blocks and other better relay mechanisms take off, the size on the wire will be less and less relevant.